lighthouse-core/report/html/report-styles.css

```diff
@@ -894,7 +894,7 @@
 }
 
 .lh-audit-group__header span.lh-audit-group__title {
-  font-weight: bold;
+  font-weight: 700;
```

Comment:
i'd rather keep this as `bold`, unless you want to change all `bold` to `700`.

Reply:
@yuinchien?

Reply:
I think you guys should decide this one, as long as the text rendered the correct weight i'm happy.

Reply:
using *bold*